File: VehicleTransport/主版本/3.开发源代码/Common/Model/DataCommand/OutDataCommand/OutGetPlanDetailModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Common.Model.DataCommand.OutDataCommand
{
    public class OutGetPlanDetailModel
    {
        public List<DB_VehicleTransportManage.Model.m_Plan> Listm_Plan = new List<DB_VehicleTransportManage.Model.m_Plan>();

        public List<DB_VehicleTransportManage.Model.m_Apply> Listm_Apply = new List<DB_VehicleTransportManage.Model.m_Apply>();

        public List<DB_VehicleTransportManage.Model.m_Plan_ApplyMaterie> Listm_Plan_ApplyMaterie = new List<DB_VehicleTransportManage.Model.m_Plan_ApplyMaterie>();

        public List<DB_VehicleTransportManage.Model.m_Plan_ApplyVehicle> Listm_Plan_ApplyVehicle = new List<DB_VehicleTransportManage.Model.m_Plan_ApplyVehicle>();

        public List<DB_VehicleTransportManage.Model.m_Plan_CheckVehicle> Listm_Plan_CheckVehicle = new List<DB_VehicleTransportManage.Model.m_Plan_CheckVehicle>();
    }
}

File: VehicleTransport/主版本/3.开发源代码/客户端/手机客户端/Last of KJ915_bestway/src/com/bestway/kj915/domain/res/plan/OutGetPlanDetailModel.java
package com.bestway.kj915.domain.res.plan;

import java.util.HashMap;

import com.bestway.kj915.domain.m_Plan;
import com.bestway.kj915.domain.res.BasicResInner;

public class OutGetPlanDetailModel extends BasicResInner {

	public Listm_Plan Listm_Plan;
	public Listm_Apply Listm_Apply;
	public Listm_Plan_ApplyMaterie Listm_Plan_ApplyMaterie;
	public Listm_Plan_ApplyVehicle Listm_Plan_ApplyVehicle;

	public Listm_Plan_CheckVehicle Listm_Plan_CheckVehicle;

	@Override
	public HashMap<String, Class> getMyDefineClass() {
		HashMap<String, Class> map = getNewEmpty();
		map.put("OutGetPlanDetailModel", OutGetPlanDetailModel.class);
		map.put("m_Plan", m_Plan.class);
		map.put("m_Apply", m_Apply.class);
		map.put("m_Plan_ApplyMaterie", m_Plan_ApplyMaterie.class);
		map.put("m_Plan_ApplyVehicle", m_Plan_ApplyVehicle.class);
		map.put("m_Plan_CheckVehicle", m_Plan_CheckVehicle.class);
		return map;
	}

	@Override
	public HashMap<String, Class> getList() {
		HashMap<String, Class> map = getNewEmpty();
		map.put("list", Listm_Plan.class);
		map.put("list1", Listm_Apply.class);
		map.put("list2", Listm_Plan_ApplyMaterie.class);
		map.put("list3", Listm_Plan_ApplyVehicle.class);
		map.put("list4", Listm_Plan_CheckVehicle.class);
		return map;
	}

}
